src/FluentHub/Views/AppSettings/MainSettingsPage.xaml

```diff
@@ -105,19 +105,19 @@
                     Content="Accounts"
                     Tag="accounts">
                     <muxc:NavigationViewItem.Icon>
-                        <muxc:ImageIcon Source="/Assets/EmojiImages/add-user-male.png" />
+                        <muxc:ImageIcon Source="/Assets/Icons/Accounts.png" />
                     </muxc:NavigationViewItem.Icon>
                 </muxc:NavigationViewItem>
 
                 <muxc:NavigationViewItem Content="Repositories" Tag="repositories">
                     <muxc:NavigationViewItem.Icon>
-                        <muxc:ImageIcon Source="/Assets/EmojiImages/repository.png" />
+                        <muxc:ImageIcon Source="/Assets/Icons/Repositories.png" />
                     </muxc:NavigationViewItem.Icon>
                 </muxc:NavigationViewItem>
 
                 <muxc:NavigationViewItem Content="Notifications" Tag="notifications">
                     <muxc:NavigationViewItem.Icon>
-                        <muxc:ImageIcon Source="/Assets/EmojiImages/appointment-reminders.png" />
+                        <muxc:ImageIcon Source="/Assets/Notifications.png" />
                     </muxc:NavigationViewItem.Icon>
```

Comment:
mistake?

Reply:
should

/Assets/Icons/Notifications.png

Reply:
Not mistake - I'm unifying the icons to only have one size and in the same folder

Reply:
but notifications.png in `/Assets/Icons/Notifications.png` not in `/Assets/Notifications.png`

Reply:
Fixed on my machine - let me push

Reply:
done